BoxRayTracer/Scene/Camera.cs

```diff
@@ -132,18 +120,31 @@ public void MoveTo(Vector newPos)
             AssertIsConsistent();
         }
 
+        public Vector GetPos()
```

Comment:
I think this can be combined with the private field to be:
`public Vector camPos { get; private set };`

Reply:
Looks like they can, yeah. I thought this might break the internal references to, for example, camPos = foo, since this^ refactors camPos from a field to a property (per my reading of the docs). However, it looks like at least within the class these references are understood by the compiler to be references to the private set().

Reply:
Okay, yeah, so it creates it as a property which can then be syntactically accessed directly [ "foo = Camera.camPos" ] , and under the hood uses the get() and set() where needed and where allowed by the access levels. Neat.